05-onchain.md

```diff
@@ -515,7 +515,7 @@ transaction.
 With a maximum standard weight of 400000, the maximum number of HTLCs that can
 be swept in a single transaction:
 
-    max_num_htlcs = (400000 - 324 - 272 - 4*53 - 2) / 413 = 966
+    max_num_htlcs = [400000 - 324 - 272 - (4 * 53) - 2] / 413 = 966
```

Comment:
Hmm, I've not seen the use of [ and ] as parentheses for a long time.  No strong opinion, it just seems unusual.  Approve the grouping of `4*53` though.

Reply:
changed to parenthesis